### ISSUE TITLE
Fill() in webgl mode causes error

### ISSUE THREAD

**Issue by brmscheiner** [processing/p5.js#1596]
Ran into this bug while trying to replicate [1544](https://github.com/processing/p5.js/issues/1544). The canvas crashes when I use the fill function in webgl mode, but the error depends on the browser (although they seem closely related).

**Chrome 53.0.2785.116:** 
`p5.js:30677 Uncaught TypeError: Cannot read property '_applyColorBlend' of undefined`

**Firefox 48.0.1:** 
`TypeError: this._renderer is undefined`

**Safari 9.1.3:** 
`TypeError: undefined is not an object (evaluating 'this._renderer._applyColorBlend')`

Here's my code:
```
function setup() {
  createCanvas(500, 500, WEBGL);
}
function draw() {
  fill(0);
}
```

I'm in OSX El Capitan 10.11.6

**Comment by lmccart** [comment]
Hm, I'm not able to reproduce...which version of p5.js are you using? 

note that you will need to draw an object in order to see the fill applied, something like this:
```javascript
function setup() {
  createCanvas(500, 500, WEBGL);
}
function draw() {
  fill(0);
  sphere(50, 50, 50)
}
```

but it shouldn't error if you haven't drawn something.

**Comment by brmscheiner** [comment]
Oooh great question, I'm on 0.5.2. I'll upgrade tomorrow and see if I can still produce the error

**Comment by lmccart** [comment]
yes this was fixed in 0.5.3
